public/js/pimcore/object/folder/search.js

```diff
@@ -458,7 +458,7 @@ pimcore.object.search = Class.create(pimcore.object.helpers.gridTabAbstract, {
             }
 
             menu.add(new Ext.menu.Item({
-                hidden: data.data.locked,
```

Comment:
do you need this change down below (L487)  in the bulk change case?

Reply:
Thank you for reviewing, yep, indeed we need a change there as well, but it could be tricky, for example, when selecting rows/element that are allowed to be deleted along the ones that are not allowed. 
Would need some more work, i have an idea and will try it out, will push the change soon

Reply:
Something like https://github.com/pimcore/admin-ui-classic-bundle/pull/530/commits/925245f2b24e54947e7ce2eabf25dbf256ae7e9a would work, i will refine and clean it a little bit tomorrow, but highly appreciate if you could review and try it

Reply:
Possible drawbacks are that the list could be too long if one selects hundreds of rows to delete, in that case it would be ideal to display the rows that would be left untouched instead (if it would be shorter) and that there are 2 confirm popup in a row (the first to warn and the second which is the former one)